Source/Core/Core/PowerPC/JitCommon/JitBase.h

```diff
@@ -17,6 +17,8 @@
 #include "Core/PowerPC/JitCommon/JitCache.h"
 #include "Core/PowerPC/PPCAnalyst.h"
 
+#include "../Jit64/rangeset.h"
```

Comment:
64-bit specifics shouldn't be included in the architecture common areas. This needs to be modified. Just leaving a comment here to make sure it's not forgotten about.